### PR TITLE
add basic uai 2018 infrastructure

### PR DESCRIPTION
**Pull request by mspector**
I'm including the jupyter notebook because UAI has a "workflow chair" who will be hopefully running the scripts, and I want to give him some materials to learn from.

### CODE REVIEW

File: venues/auai.org/UAI/2018/process/recruitSPCProcess.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      //if a user is in the reviewers group, remove them from that group and add them to the reviewers-declined group
+      or3client.removeGroupMember('auai.org/UAI/2018/Senior_Program_Committee', note.content.email, token)
+      or3client.addGroupMember('auai.org/UAI/2018/Senior_Program_Committee/Declined', note.content.email, token)
+      .then(result => done())
+      .catch(error => console.log(error));
+    }
+    return true;
+  } else {
+    console.log('Invalid key', note.content.key);
+    return false;
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/process/recruitSPCProcess.js]: you are not calling the done function, the function never ends and a timeout error is thrown

File: venues/auai.org/UAI/2018/process/recruitSPCProcess.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+function() {
+
+  var or3client = lib.or3client;
+  var hashKey = or3client.createHash(note.content.email, "2810398440804348173");
+  if(hashKey == note.content.key) {
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/process/recruitSPCProcess.js]: trim the keys if they have spaces

File: venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<html>
```
Comment (melisabok) [inline venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.html]: shouldn't webfields be js files? using the code that @zbialecki wrote in OpenReview?

Comment (zbialecki) [reply venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.html]: Yeah, this one could easily be re-written with the functions from the webfield module.

Comment (melisabok) [reply venues/auai.org/UAI/2018/webfield/recruitSPCWebfield.html]: Great, we should do that and don't propagate old code.

**Pull request closed without merging**